tests/test_2024_10_01_holesky.py

```diff
@@ -0,0 +1,226 @@
+"""
+Tests for voting 01/10/2024
+
+"""
+
+from scripts.vote_2024_10_01_holesky import start_vote
+from brownie import interface, ZERO_ADDRESS, reverts, web3, accounts, convert
+from utils.test.tx_tracing_helpers import *
+from utils.voting import find_metadata_by_vote_id
+from utils.ipfs import get_lido_vote_cid_from_str
+from utils.config import contracts, LDO_HOLDER_ADDRESS_FOR_TESTS, network_name
+from utils.easy_track import create_permissions
+from configs.config_mainnet import (
+    DAI_TOKEN,
+    USDC_TOKEN,
+    USDT_TOKEN,
+)
+from utils.test.easy_track_helpers import create_and_enact_payment_motion, check_add_and_remove_recipient_with_voting
+from utils.test.event_validators.easy_track import (
+    validate_evmscript_factory_added_event,
+    EVMScriptFactoryAdded,
+    validate_evmscript_factory_removed_event,
+)
+_
+# STETH_TRANSFER_MAX_DELTA = 2
+
+def test_vote(helpers, accounts, vote_ids_from_env, stranger, ldo_holder, bypass_events_decoding):
+    steth = contracts.lido
+    easy_track = contracts.easy_track
+
+    evm_script_factories_before = easy_track.getEVMScriptFactories()
+
+    alliance_ops_top_up_evm_script_factory_new = "0x343fa5f0c79277e2d27e440f40420d619f962a23" # TopUpAllowedRecipients
+    assert alliance_ops_top_up_evm_script_factory_new not in evm_script_factories_before
+
+    alliance_multisig_acc = accounts.at("xd4090CA1134F8dE1450B8246916F73d212efdEf6", force=True) # Testnet DAO Multisigs
+    alliance_dai_usdc_contract = accounts.at("0x2eb8e9198e647f80ccf62a5e291bcd4a5a3ca68c", force=True) # Testnet Stablecoins
```

Comment:
it's rudiments from the previous launch connected with swaps. 
We can't use these addresses here. you have to use alliance_multisig_acc as a receiver